Max/src/main/java/springbootApp/java/exceptions/InstrumentoDuplicadoException.java

```diff
@@ -0,0 +1,7 @@
+package springbootApp.java.exceptions;
+
+public class InstrumentoDuplicadoException extends RuntimeException {
+    public InstrumentoDuplicadoException(String message) {
+        super(message);
+    }
+}
```

Comment:
Yo haria que extienda de Exception mas que de RuntimeException, al ser una CustomException el caso del duplicado tiene mas sentido a mi lectura. Una runtime es una exception en tiempo de ejecucion no "checkeada" y aca la estamos manejando, por lo que lo mas correcto seria de la clase Exception.

```suggestion
public class InstrumentoDuplicadoException extends Exception {
    public InstrumentoDuplicadoException(String message) {
        super(message);
    }
}
```